bundles/mapping/layerswipe/handler/SwipeToolHandler.js

```diff
@@ -0,0 +1,65 @@
+import { StateHandler, controllerMixin } from 'oskari-ui/util';
+
+class UIHandler extends StateHandler {
+    constructor (tool) {
+        super();
+        this.tool = tool;
+        this.sandbox = tool.getSandbox();
+        this.setState({
+            autoStart: false,
+            noUI: false
+        });
+    };
+
+    init (pluginConfig) {
+        this.updateState({
+            ...pluginConfig.conf,
+            autoStart: pluginConfig?.state?.active,
+            noUI: pluginConfig?.conf?.noUI
+        });
+    }
+
+    syncState () {
```

Comment:
We probably don't need this anymore? Check if it can be removed.

Reply:
we do, unfortunately. The plugin might not exist yet where we could otherwise set the state, so this is needed for the ui state & the plugin state to stay 'n sync.